spec/helpers/people_filter_helper_spec.rb

```diff
@@ -62,4 +62,101 @@
       expect(people_filter_attribute_value("company", "false")).to eq "nein"
     end
   end
+
+  describe "#people_filter_filter_options" do
+    describe "qualifications" do
+      let(:filter) do
+        Person::Filter::Qualification.new("qualification", attrs)
+      end
+
+      let(:selected_kind_array) { [QualificationKind.first.id, QualificationKind.second.id] }
+
+      let(:attrs) do
+        {
+          qualification_kind_ids: selected_kind_array,
+          validity: "all",
+          match: "one"
+        }
+      end
+
+      it "returns qualification kind options with selected" do
+        html = helper.people_filter_qualification_kind_options(filter)
+        QualificationKind.list.without_deleted.each do |kind|
```

Comment:
am besten in ein `let(:kinds) { QualificationKind.list.without_deleted }` extrahieren